README.md

```diff
@@ -143,6 +143,7 @@ variables:
   This indicates the desired file system type to use, e.g.: "xfs", "ext4", "swap".
   The default is determined according to the OS and release
   (currently `xfs` for all the supported systems).
+  Use "unformatted" if you do not want file system to be present.
```

Comment:
or maybe this should be `"unconfigured"`?

Reply:
Not sure how we note destructive operations elsewhere, but seems to me like we should say something like `**WARNING**: Using "unconfigured" on an existing filesystem will remove the filesystem.`

Reply:
Thanks. The correct value is actually 'unformatted' in the code. I made sure the value is unified.
I have also added a warning note into the README.